projects/commudle-admin/src/app/feature-modules/labs/components/lab-display-card/lab-display-card.component.scss

```diff
@@ -0,0 +1,109 @@
+@import '/projects/commudle-admin/src/assets/styles/variables.scss';
+.lab-display-card {
+    width: 280px;
+    height: 264px;
+    border: 1px solid #72C1FF;
+    border-radius: 5px;
+
+    p, div, h4 {
+        font-family: $primary-font-family;
+        margin: 0px; //to remove default margin
+    }
+    .header-image {
+        width: 100%;
+        height: 134px;
+        overflow: hidden;
+        img {
+            width: 100%;
+        }
+        border-bottom: 1px solid #72C1FF;
+    }
+
+    .content {
+        padding: 12px 16px 8px;
+        display: flex;
+        flex-direction: column; 
+
+        .title {
+            a {
+                text-decoration: none;
+            }
+            h4 {
+                font-size: 18px;
+                color: rgba(83, 110, 255, 1);
+                line-height: 21px;
+            }
+
+            p {
+                font-style: italic;
+                font-weight: normal;
+                font-size: 12px;
+                line-height: 14px;
+                color: rgba(47, 46, 65, 0.5);
+                padding-top:4px;
+            }
+        }
+
+        .author-stats {
+
+
+            padding-top: 24px;
+            
+            .author {
+                display: flex;
+                flex-direction: row;
+                .image {
+                    width: 34px;
+                    height: 34px;
+                    img {
+                        width: 100%;
+                        border-radius: 50%;
+                    }
+                }
+
+                .details {
+                    list-style: 3px;
```

Comment:
I don't think this is a valid property, please check

Reply:
Plus, indentation issues